app/dbCache.rb

```diff
@@ -212,32 +213,15 @@ def signalDbRefill
   Process.kill("WINCH", -Process.getpgrp)
 end
 
-def getLastTableUpdateTime
-  utime = nil
-  DB.fetch("SHOW TABLE STATUS WHERE Name in ('units', 'unit_hier', 'redirects')").each { |row|
-    if row[:Update_time] && (!utime || row[:Update_time] > utime)
-      utime = row[:Update_time]
-    end
-  }
-  return utime
-end
-
 ##################################################################################################
 # Database caches for speed. We check every 30 seconds for changes. These tables change infrequently.
 
 Thread.new {
   begin
-    prevTime = getLastTableUpdateTime
+    puts "IN DB CACHE Thread"
```

Comment:
leftover debug logging?

Reply:
I'd suggest changing this text to something slightly more meaningful if we want to keep it, such as "ENGAGED: 60 minute DB cache refill timer" or similar.